Frontend/CO.CDP.OrganisationApp/Pages/Consortium/ConsortiumOverview.cshtml

```diff
@@ -0,0 +1,106 @@
+    var ppon = (organisationDetails.Identifier.Scheme == OrganisationSchemeType.Other
+                        ? StaticTextResource.Consortium_ConsortiumOverview_Ppon_Status_Pending
+                        : organisationDetails.Identifier.Id);
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full govuk-body">
+        <span class="govuk-caption-l">
+            PPON: @ppon
+        </span>
+        <h1 class="govuk-heading-l">
+            @organisationDetails.Name
+        </h1>
+
+        <h2 class="govuk-heading-m">@StaticTextResource.Consortium_ConsortiumOverview_Heading</h2>
+
+        <dl class="govuk-summary-list">
+
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">Name</dt>
+                <dd class="govuk-summary-list__value">
+                    <p class="govuk-body">@organisationDetails.Name</p>
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link govuk-link--no-visited-state" href="#">@StaticTextResource.Global_Change<span class="govuk-visually-hidden"> @StaticTextResource.Consortium_ConsortiumOverview_HiddenSpan_name</span></a>
+                </dd>
+            </div>
+
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">@StaticTextResource.Consortium_ConsortiumOverview_Address_list</dt>
+                <dd class="govuk-summary-list__value">
+                    <p class="govuk-body">@postalAddress!.StreetAddress</p>
+                    <p class="govuk-body">@postalAddress.Locality</p>
+                    @if (!string.IsNullOrWhiteSpace(postalAddress.Region))
+                    {
+                        <p class="govuk-body">@postalAddress.Region</p>
+                    }
+                    <p class="govuk-body">@postalAddress.PostalCode</p>
+                    <p class="govuk-body">@postalAddress.CountryName</p>
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link govuk-link--no-visited-state" href="#">@StaticTextResource.Global_Change<span class="govuk-visually-hidden"> @StaticTextResource.Consortium_ConsortiumOverview_HiddenSpan_address</span>
+                    </a>
+                </dd>
+            </div>
+
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">@StaticTextResource.Consortium_ConsortiumOverview_EmailAddress_list</dt>
+                <dd class="govuk-summary-list__value">
+                    <p class="govuk-body">@organisationDetails.ContactPoint.Email</p>
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link govuk-link--no-visited-state" href="#">@StaticTextResource.Global_Change<span class="govuk-visually-hidden"> @StaticTextResource.Consortium_ConsortiumOverview_HiddenSpan_emailaddress</span></a>
```

Comment:
Please could the word "change" be including in the whole key for this? See `SupplierToBuyer_OrganisationSummary_ChangeDevolvedRegions` for an example. Concatenating the word "Change" onto the beginning of the sentence may not always work in welsh - it needs to make sense in context

Reply:
Done